Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/RotationManipulators.cpp

```diff
@@ -13,45 +13,74 @@
 namespace AzToolsFramework
 {
     RotationManipulators::RotationManipulators(const AZ::Transform& worldFromLocal)
+        : m_viewAngularManipulator { AngularManipulator::MakeShared(worldFromLocal) }
     {
+        const auto saveMostRecentActionCb = [this](const AngularManipulator::Action& action)
+        {
+            m_angularManipulatorFeedback.m_mostRecentAction = action;
+        };
+
         for (size_t manipulatorIndex = 0; manipulatorIndex < m_localAngularManipulators.size(); ++manipulatorIndex)
         {
-            m_localAngularManipulators[manipulatorIndex] = AngularManipulator::MakeShared(worldFromLocal);
+            auto angularManipulator = AngularManipulator::MakeShared(worldFromLocal);
+            angularManipulator->InstallLeftMouseDownCallback(saveMostRecentActionCb);
+            angularManipulator->InstallLeftMouseUpCallback(saveMostRecentActionCb);
+            angularManipulator->InstallMouseMoveCallback(saveMostRecentActionCb);
+            m_localAngularManipulators[manipulatorIndex] = angularManipulator;
         }
 
-        m_viewAngularManipulator = AngularManipulator::MakeShared(worldFromLocal);
-
         m_manipulatorSpaceWithLocalTransform.SetSpace(worldFromLocal);
+        m_viewAngularManipulator->InstallLeftMouseDownCallback(saveMostRecentActionCb);
+        m_viewAngularManipulator->InstallLeftMouseUpCallback(saveMostRecentActionCb);
+        m_viewAngularManipulator->InstallMouseMoveCallback(saveMostRecentActionCb);
     }
 
     void RotationManipulators::InstallLeftMouseDownCallback(const AngularManipulator::MouseActionCallback& onMouseDownCallback)
     {
+        const auto saveMostRecentActionHookCb = [this, onMouseDownCallback](const AngularManipulator::Action& action)
+        {
+            m_angularManipulatorFeedback.m_mostRecentAction = action;
+            onMouseDownCallback(action);
+        };
+
         for (AZStd::shared_ptr<AngularManipulator>& manipulator : m_localAngularManipulators)
         {
-            manipulator->InstallLeftMouseDownCallback(onMouseDownCallback);
+            manipulator->InstallLeftMouseDownCallback(saveMostRecentActionHookCb);
```

Comment:
Nitpick - maybe it's just me, but this naming seems a tiny bit unclear. The callback is doing two things - saving the most recent action, and then doing the mouse down behaviour, but the name only communicates the first of those things to me, and kind of implies that saving the action is the most important thing that's happening here. I'd also say it's a little bit confusing to use the same name for other lambdas later on which do different things.

Reply:
Ok I'll make it more descriptive 👌